enterprise/api/v1/serializers.py

```diff
@@ -97,3 +97,27 @@ class Meta:
     user = UserSerializer()
     enterprise_customer = EnterpriseCustomerSerializer()
     data_sharing_consent = UserDataSharingConsentAuditSerializer(many=True)
+
+
+class EnterpriseCustomerUserEntitlementSerializer(serializers.Serializer):
```

Comment:
@mattdrayer read/write capabilities are part of views like `EnterpriseReadOnlyModelViewSet` but for this serializer we do not have a view as this serializer is only being used for a `detail_route`.